### PR TITLE
add support article for mysql ssl connections

### PR DESCRIPTION
**Pull request by wcpines**
May need some cleaning up.

### CODE REVIEW

File: source/topics/troubleshooting/mysql-connection-refused.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+If you run `aptible db:tunnel $DB_HANDLE` and try connecting to the db with the `mysql` commandline client, you may see the folowing error:
+
+      ERROR 1045 (28000): Access denied for user 'aptible'@'ip-[IP_ADDRESS].ec2.internal' (using password: YES)
+
+On Aptible, MySQL servers are configured to require SSL for any TCP connection from the user `aptible`, but the client does not connect over ssl by default, resulting in an error.
```
Comment (fancyremarker) [inline source/topics/troubleshooting/mysql-connection-refused.md]: Use consistent uppercase for "SSL." (Here and below.)

File: source/topics/troubleshooting/mysql-connection-refused.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+To verify your connection is running over ssl run
+
+    mysql> show status like 'Ssl_cipher';
+    +---------------+--------------------+
+    | Variable_name | Value              |
+    +---------------+--------------------+
+    | Ssl_cipher    | DHE-RSA-AES256-SHA |
+    +---------------+--------------------+
+
+  or
```
Comment (fancyremarker) [inline source/topics/troubleshooting/mysql-connection-refused.md]: I'd omit this part.

File: source/topics/troubleshooting/mysql-connection-refused.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+If you run `aptible db:tunnel $DB_HANDLE` and try connecting to the db with the `mysql` commandline client, you may see the folowing error:
```
Comment (fancyremarker) [inline source/topics/troubleshooting/mysql-connection-refused.md]: This applies to both `aptible db:tunnel` but also general connections from an app to an Aptible MySQL database, so I'd mention both cases.

**Comment by fancyremarker** [comment]
This is a good start! You'll also need to add an entry to `data/topics.yml`. See 09b4c4e for an example. I think this belongs in the "Troubleshooting" section.

File: source/topics/troubleshooting/mysql-connection-refused.md (hunk 2 above)
Comment (wcpines) [inline source/topics/troubleshooting/mysql-connection-refused.md]: Assuming you mean 9-18?

Comment (fancyremarker) [inline source/topics/troubleshooting/mysql-connection-refused.md]: I meant 18-29.

**Comment by wcpines** [comment]
:+1:

**Comment by fancyremarker** [comment]
Thanks @wcpines !

**Pull request merged**